### ISSUE TITLE
Undefined variable error with Texplore and g:netrw_browse_split = 3

### ISSUE THREAD

**Issue by harjoc** [vim/vim#1311]
I have this in my .vimrc, so F3 opens the explorer in a new tab, and opened files will also go to a new tab:

```vim
map <F3> :Texplore<CR>
let g:netrw_browse_split = 3
```

When I select a file from the explorer with Enter, I get these messages:

```
Error detected while processing function <SNR>20_NetrwBrowseChgDir:
line  135:
E121: Undefined variable: b:netrw_curdir
```
And this is displayed for a second:
```
Invalid arguments for function SetRexDir
```

Saw this after upgrade to Ubuntu 16.04 (vim 7.4.1689-3ubuntu1.2). The error didn't show with vim 7.4.052-1ubuntu3.1 on Ubuntu-14.04.

**Comment by archtan** [comment]
It appears that this issue has been fixed in [commit aa3b15db](https://github.com/vim/vim/commit/aa3b15dbebf333282503d6031e2f9ba6ee4398ed#diff-3abb9629535900f0c07a312231e18483L4697), which was part of the [7.4.1761 release](https://github.com/vim/vim/releases/tag/v7.4.1761).

`7.4.1689-3ubuntu1.2` is still the latest version in Ubuntu 16.04, so it might be best to file a bug in the Ubuntu tracker: https://bugs.launchpad.net/ubuntu/

I've simply patched my `/usr/share/vim/vim74/autoload/netrw.vim`.  For my purpose, I found it was sufficient to change `b:netrw_curdir` to just `curdir` in `call s:SetRexDir(a:islocal,b:netrw_curdir)` (line 4701) -- but I have added in the new `if` block as well, since I don't really know what I'm doing when it comes to Vim scripts!

**Comment by cecamp** [comment]
archtan wrote:
>
> It appears that this issue has been fixed in commit aa3b15db
> <https://github.com/vim/vim/commit/aa3b15dbebf333282503d6031e2f9ba6ee4398ed#diff-3abb9629535900f0c07a312231e18483L4697>,
> which was part of the 7.4.1761 release
> <https://github.com/vim/vim/releases/tag/v7.4.1761>.
>
> |7.4.1689-3ubuntu1.2| is still the latest version in Ubuntu 16.04, so
> it might be best to file a bug in the Ubuntu tracker:
> https://bugs.launchpad.net/ubuntu/
>
> I've simply patched my |/usr/share/vim/vim74/autoload/netrw.vim|. For
> my purpose, I found it was sufficient to change |b:netrw_curdir| to
> just |curdir| in |call s:SetRexDir(a:islocal,b:netrw_curdir)| (line
> 4701) -- but I have added in the new |if| block as well, since I don't
> really know what I'm doing when it comes to Vim scripts!
>
>
Hello:

As a suggestion: if you have a problem with netrw, its a good idea to
get the latest version from
http://www.drchip.org/astronaut/vim/index.html#NETRW to see if the
problem has been addressed already.

Regards,
Chip Campbell

**Comment by lacygoill** [comment]
> It appears that this issue has been fixed in commit aa3b15db, which was part of the 7.4.1761 release.

So the issue is completely fixed as of today, right?
